core/storage/statistics/gae_models.py

```diff
@@ -82,6 +82,10 @@ class AnswerSubmittedEventLogEntryModel(base_models.BaseModel):
     time_spent_in_state_secs = ndb.FloatProperty()
     # Whether the submitted answer received useful feedback
     is_feedback_useful = ndb.BooleanProperty(indexed=True)
+    # The version of the event schema used to describe an event of this type.
+    # Details on the schema are given in the docstring for this class.
```

Comment:
Are they? I don't see them.

It might be fine to have the details you're talking about in feconf.py if they're not class-specific.

Ditto elsewhere.

Reply:
Removed it. Thanks for pointing out. IT might be better to have a doctoring in feconf.